src/reader_async.rs

```diff
@@ -131,6 +131,11 @@ impl ParquetFile {
         Ok(self.meta.metadata().as_ref().to_owned().into())
     }
 
+    #[wasm_bindgen]
+    pub fn schema(&self) -> WasmResult<arrow_wasm::Schema> {
+        Ok(self.meta.schema().as_ref().to_owned().into())
```

Comment:
Can you do `.clone()` instead of `as_ref().to_owned()`?

Reply:
If `.schema()` gives you an `&Arc<Schema>`, then `.clone()` directly will just increase the reference count, while `.as_ref().to_owned()` will I think create a new `Schema`, copying all the underlying field data